server/openapi.yaml

```diff
@@ -737,6 +737,42 @@ paths:
           $ref: "#/components/responses/NotFoundError"
         "500":
           $ref: "#/components/responses/InternalError"
+  /display/set_resolution:
+    post:
+      summary: Safely set the X display resolution
+      operationId: setResolution
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/SetResolutionRequest"
+      responses:
+        "200":
+          description: Resolution updated
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/OkResponse"
+        "409":
+          $ref: "#/components/responses/ConflictError"
+        "400":
+          $ref: "#/components/responses/BadRequestError"
+        "500":
+          $ref: "#/components/responses/InternalError"
+  /display/status:
+    get:
+      summary: Report whether resize is currently safe
+      operationId: displayStatus
+      responses:
+        "200":
+          description: Status
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/DisplayStatus"
+        "500":
+          $ref: "#/components/responses/InternalError"
```

Comment:
I think we should probably just nix this. From an API perspective we should just try to update the display directly over a `get -> set`